src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const SEX_MALE = 'm';
+  const isWifeOrHusband = sex === SEX_MALE
```

Comment:
I suppose that in this kind of variable should be stored some boolean value
If you need to store the marriage data, will be better to use the name according to this statement
`marriageInfo` will be better name